assets/src/components/app.tsx

```diff
@@ -38,7 +39,9 @@ const AppRoutes = () => {
     openView === OpenView.Late || location.pathname === "/"
 
   const routesForVehicles = featureIsEnabled("presets_workspaces")
-    ? currentRouteTab(routeTabs)?.selectedRouteIds || []
+    ? flatten(
```

Comment:
**suggestion (non-blocking):** Would you want to apply `uniq` to this list in addition to `flatten` in case the user has a route open in multiple tabs? I don't know whether that would actually matter or not in practice.

Reply:
Oh, that's actually a really good point - each route gets turned into a subscription to the relevant channel, and weird things can happen when you double-subscribe to a single channel - see "Duplicate Join Subscriptions" on [this page](https://hexdocs.pm/phoenix/js/). Will update.